### PR TITLE
rescues from Octokit::NotFound and returns the user with a flash

### PR DESCRIPTION
**Pull request by wadtech**
The simplest approach, don't do anything punishing (like a full 404)

A future idea might be to do a fuzzy search based on similar projects and do a 'did you mean' style thing.

### CODE REVIEW

**Comment by wadtech** [comment]
for #50

File: app/controllers/projects_controller.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
+require 'octokit'
```
Comment (andrew) [inline app/controllers/projects_controller.rb]: This shouldn't need to be here, it's in the Gemfile so should be automatically required by bundler on startup

**Comment by andrew** [comment]
Looks like just what I would have done, thanks @wadtech, fixes #50

**Comment by wadtech** [comment]
Cool I'll merge this then

**Comment by andrew** [comment]
:+1:

**Pull request merged**